tests/Exceptionless.Tests/AppWebHostFactory.cs

```diff
@@ -1,12 +1,37 @@
+using Aspire.Hosting;
+using Aspire.Hosting.ApplicationModel;
 using Exceptionless.Insulation.Configuration;
 using Exceptionless.Web;
 using Microsoft.AspNetCore.Mvc.Testing;
 using Microsoft.AspNetCore.TestHost;
+using Xunit;
 
 namespace Exceptionless.Tests;
 
-public class AppWebHostFactory : WebApplicationFactory<Startup>
+public class AppWebHostFactory : WebApplicationFactory<Startup>, IAsyncLifetime
 {
+    private DistributedApplication? _app;
+
+    public DistributedApplication App => _app ?? throw new InvalidOperationException("The application is not initialized");
+
+    public Task InitializeAsync()
+    {
+        var options = new DistributedApplicationOptions { AssemblyName = typeof(ElasticsearchResource).Assembly.FullName, DisableDashboard = true };
+        var builder = DistributedApplication.CreateBuilder(options);
+
+        // don't use random ports for tests
+        builder.Configuration["DcpPublisher:RandomizePorts"] = "false";
+
+        builder.AddElasticsearch("Elasticsearch", port: 9200)
+            .WithContainerName("Exceptionless-Elasticsearch-Test")
+            .WithImageTag("8.17.0")
```

Comment:
would be nice to have all these versions in one spot.

Reply:
Changed to use default so the version is only in one place.